test/gateway_integration/suite_test.go

```diff
@@ -93,8 +92,8 @@ var _ = BeforeSuite(func() {
 	Expect(err).NotTo(HaveOccurred())
 	Expect(cfg).NotTo(BeNil())
 
-	err = v1alpha1.AddToScheme(scheme.Scheme)
-	Expect(err).NotTo(HaveOccurred())
+	//err = v1alpha1.AddToScheme(scheme.Scheme)
```

Comment:
do we just want to remove if not needed?

Reply:
yep, can be removed

Reply:
there is no api in this repo now, just left over from when i was working through it.